lib/storage/index_db.go

```diff
@@ -1345,7 +1345,7 @@ func (is *indexSearch) getTSDBStatusWithFiltersForDate(tfss []*TagFilters, date
 	if len(tfss) > 0 {
 		tr := TimeRange{
 			MinTimestamp: int64(date) * msecPerDay,
-			MaxTimestamp: int64(date+1) * msecPerDay - 1,
+			MaxTimestamp: int64(date+1)*msecPerDay - 1,
```

Comment:
seems, that this change out of scope for this PR

Reply:
Linter fails without it. Do you think I should move https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2665/commits/86fcdde07c9819e4bc6c81ba62a0de86ef0e1bca to separate PR?

Reply:
meh, @valyala have to use the same linter